src/dash/models/DashManifestModel.js

```diff
@@ -279,24 +279,6 @@ function DashManifestModel() {
         return isDynamic;
     }
 
-    function hasProfile(manifest, profile) {
-        var has = false;
-
-        if (manifest.profiles && manifest.profiles.length > 0) {
-            has = (manifest.profiles.indexOf(profile) !== -1);
-        }
-
-        return has;
-    }
-
-    function getIsOnDemand(manifest) {
-        return hasProfile(manifest, 'urn:mpeg:dash:profile:isoff-on-demand:2011');
-    }
-
-    function getIsDVB(manifest) {
```

Comment:
This is called from https://github.com/Dash-Industry-Forum/dash.js/blob/development/src/streaming/utils/BaseURLSelector.js#L79